### PR TITLE
[Fix] Supplying the model for lyft trained by pointpillars

### PR DESCRIPTION
**Pull request by wHao-Wu**
* Supplying the model for lyft trained by pointpillars with FPN architecture
* Supplying the model for lyft trained by pointpillars with SECFPN architecture

### CODE REVIEW

**Review by THU17cyz — APPROVED** [review]
Checked that the links are valid.

**Review by Tai-Wang — APPROVED** [review]

**Review by Tai-Wang — CHANGES_REQUESTED** [review]
Please also modify the corresponding benchmarks in other config folders like [regnet](https://github.com/open-mmlab/mmdetection3d/tree/master/configs/regnet) and [ssn](https://github.com/open-mmlab/mmdetection3d/tree/master/configs/ssn).

**Comment by wHao-Wu** [comment]
> Please also modify the corresponding benchmarks in other config folders like [regnet](https://github.com/open-mmlab/mmdetection3d/tree/master/configs/regnet) and [ssn](https://github.com/open-mmlab/mmdetection3d/tree/master/configs/ssn).

Thanks for your advice. Regnet and ssn model on Lyft is still in progress. I will release those models after completion.

File: configs/pointpillars/README.md [hunk 1]
```diff
@@ -37,8 +37,8 @@ We implement PointPillars and provide the results and checkpoints on KITTI, nuSc
 
 |  Backbone   | Lr schd | Mem (GB) | Inf time (fps) | Private Score | Public Score | Download |
 | :---------: | :-----: | :------: | :------------: | :----: |:----: | :------: |
-|[SECFPN](./hv_pointpillars_secfpn_sbn-all_4x8_2x_lyft-3d.py)|2x|||13.4|13.4||
-|[FPN](./hv_pointpillars_fpn_sbn-all_4x8_2x_lyft-3d.py)|2x|||14.0|14.2||
+|[SECFPN](./hv_pointpillars_secfpn_sbn-all_2x8_2x_lyft-3d.py)|2x|12.2||13.9|14.1|[model](https://download.openmmlab.com/mmdetection3d/v0.1.0_models/pointpillars/hv_pointpillars_secfpn_sbn-all_2x8_2x_lyft-3d/hv_pointpillars_secfpn_sbn-all_2x8_2x_lyft-3d_20210517_204807-2518e3de.pth) &#124; [log](https://download.openmmlab.com/mmdetection3d/v0.1.0_models/pointpillars/hv_pointpillars_secfpn_sbn-all_2x8_2x_lyft-3d/hv_pointpillars_secfpn_sbn-all_2x8_2x_lyft-3d_20210517_204807.log.json)|
```
Comment (ZwwWayne) [inline configs/pointpillars/README.md:40]: Should update metafile.

Comment (wHao-Wu) [reply configs/pointpillars/README.md:40]: @ZwwWayne 

Thanks for your advice. I have updated the metafile in pointpillars.

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**